gradle-mvn-push.gradle

```diff
@@ -0,0 +1,104 @@
+apply plugin: 'maven-publish'
+apply plugin: 'signing'
+
+final isSnapshot = version.contains('SNAPSHOT')
+
+final sharedPublicationConfiguration(publication) {
+    publication.pom {
+        url = 'https://randoop.github.io/randoop/'
+        developers {
+            // These are the lead developers, not all the contributors.
+            developer {
+                id = 'mernst'
+                name = 'Michael Ernst'
+                email = 'mernst@cs.washington.edu'
+                url = 'https://homes.cs.washington.edu/~mernst/'
+                organization = 'University of Washington'
+                organizationUrl = 'https://www.cs.washington.edu/'
+            }
+        }
+
+        scm {
+            url = 'https://github.com/randoop/randoop'
+            connection = 'scm:git:git://github.com/randoop/randoop.git'
+            developerConnection = 'scm:git:ssh://git@github.com/randoop/randoop.git'
+        }
+
+        licenses {
+            license {
+                name = 'The MIT License'
+                url = 'http://opensource.org/licenses/MIT'
+                distribution = 'repo'
+            }
+            // TODO:
```

Comment:
What license does this use?

Reply:
I can't find a short name for it.  That short name might not be useful, because Oracle keeps changing their license terms for a given-named license.  It is not restrictive.  It seems close to [Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)](https://creativecommons.org/licenses/by-sa/4.0/?ref=chooser-v1)3.

Reply:
So should I add a license block with that links to the license you provided?  It's not clear from your comment.

Reply:
Sorry for the ambiguity.  I perhaps didn't understand your question.  If you want to include a license, that is the best one to include.  If so, I would add a comment saying it's close to that and including a link to the actual license for clarity.

Reply:
I copied this line from `randoop/license.txt` and beyond that I don't have any insight into this file or its license. Your comment said I could add one if I want, but I don't know if we need one or not.

Reply:
Can you clarify what you mean by "this file"?  Are you asking about the license for `gradle-mvn-push.gradle`?

Regarding whether the gradle file needs to have a license block, that seems to be a gradle question.  I believe that some people won't use a project that lacks one.

Reply:
Here's the text of randoops `license.txt`

```
Unless otherwise noted, Randoop is distributed under the MIT license,
which appears below.  Some files in Randoop's distribution are
distributed under a different license. These exceptions are:

Randoop source code:
  src/randoop/util/StreamRedirectThread.java (License in file)

Experimental subject programs (not part of Randoop's implementation)
  systemTests/ (various licenses)
  tests/ (various licenses)
```

I've added a license block for The MIT License.  I don't know if we need one for `src/randoop/util/StreamRedirectThread.java`.  It's in the Jar file, so I assume so. We are not distributing the experimental subject programs in Maven, so we don't have to worry about them.

Reply:
The file is not used, so I will open a pull request to remove it from Randoop, which makes this question moot.